charts/postgres-db-sync/table2table_sync.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+date;
+
+echo "Synchronize ${INPUT_DB_HOST}:${INPUT_SCHEMA}.\"${INPUT_TABLE}\"";
+echo "to ${OUTPUT_DB_NAME}:${OUTPUT_SCHEMA}.\"${OUTPUT_TABLE}\"";
+
+# Output DB Connection
+export PGHOST=${OUTPUT_DB_HOST} PGDATABASE=${OUTPUT_DB_NAME} PGUSER=${OUTPUT_DB_USER} PGPASSWORD=${OUTPUT_DB_PASS};
+
+# Truncate table if required
+if [[ "$TRUNCATE_TARGET_TABLE" == "true" ]]
+then
+  echo "Truncating ${OUTPUT_DB_NAME}:${OUTPUT_SCHEMA}.\"${OUTPUT_TABLE}";
+  psql  -c "TRUNCATE TABLE ${OUTPUT_SCHEMA}.\"${OUTPUT_TABLE}\";";
+fi
+
+# Find the oldest record in the output table so that we can export all the
+# newer records than this
+export OLDEST_RECORD=`(psql --tuples-only -c " \
+  SELECT COALESCE(MAX(\"${TIMESTAMP_COLUMN}\"),'2020-01-01 00:00:00'::timestamp) \
```

Comment:
I don't understand why we have the `COALESCE`. If we don't have any date, we can remove the where clause below